src/styles/blocks.scss/calendar.scss

```diff
@@ -0,0 +1,18 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: $calendar-width;
+}
+
+@each $day, $margin-left in $week-day {
+  .calendar--start-#{$day} .calendar__day:first-child {
+    margin-left: $margin-left;
+  }
+}
+
+@for $date from 28 through 31 {
+  .calendar--length-#{$date} .calendar__day:nth-child(n + #{$date + 1}) {
+    display: none;
+  }
+}
```

Comment:
You could write it using nesting, in the `calendar` selector. The same situation with the` calendar__day` and `for `loop.